apps/examples/testcase/le_tc/network/network_tc_main.c

```diff
@@ -37,9 +49,18 @@ int tc_network_main(int argc, char *argv[])
 		return ERROR;
 	}
 
+	g_tc_fd = open(KERNEL_TEST_DRVPATH, O_WRONLY);
```

Comment:
Missing close for this fd at end of tc_network_main

Reply:
corrected